src/HTSPDemuxer.cpp

```diff
@@ -454,82 +458,80 @@ void CHTSPDemuxer::ParseSubscriptionStart ( htsmsg_t *m )
 
     /* Find stream */
     m_streamStat[idx] = 0;
+    PVR_STREAM_PROPERTIES::PVR_STREAM stream;
```

Comment:
No (in this case), as you did not specify an initilaizer. => http://stackoverflow.com/questions/1069621/are-members-of-a-c-struct-initialized-to-0-by-default